docs/content/dependencies.md

```diff
@@ -18,24 +18,23 @@ are!
 Briefly, Converge operates by thinking about your deployment as a graph of tasks
 that need to be done. It walks around the graph from the leaves (AKA tasks with
 no dependencies) all the way to to the root (AKA Rome, where all roads lead.)
-Let's explore what that means with one of our graphs from before:
+Let's explore what that means with a graph from the [params example in the
+getting started]({{< ref "getting-started.md" >}}#params) guide:
 
 {{< figure src="/images/getting-started/hello-world-params.png"
            caption="A graph with a parameter. The file hello.txt depends on the name parameter." >}}
 
-What does Converge do when you ask it to apply this graph? When Converge loads
-this file, it will load then file and then start walking at the node that
-doesn't have any dependencies. In this case, that's `param.name = "World"`. When
-`param.name` has been successfully walked, it will move on to `File: hello.txt`.
-If we're successful, the root (`/`) will be marked as successful, and our graph
-will be successful. Neat!
+What does Converge do when you ask it to apply this graph? Converge will load
+the file and start walking at a node that has no dependencies. In this case,
+that's `param.name = "World"`. When `param.name` has been successfully walked,
+Converge will move on to `File: hello.txt`, and then to the root (`/`). Once
+each node is marked as successful, our graph walk is complete. Neat!
```

Comment:
I lean more toward taking this out. You were referring to entire Graph Walking section, right?